apps/namadillo/src/atoms/balance/atoms.ts

```diff
@@ -107,7 +107,7 @@ export const viewingKeysAtom = atomWithQuery<
 });
 
 export const storageShieldedBalanceAtom = atomWithStorage<
-  Record<Address, { address: string; minDenomAmount: BigNumber }[]>
+  Record<Address, { address: Address; minDenomAmount: string }[]>
```

Comment:
I think we should store amount as BigNumber and save unnecessary conversions later

Reply:
@pedrorezende I agree, but this is the storage, meaning it's saving on LocalStorage as a string, so, unfortunately, we can't save it as a BigNumber

I found this issue because we are now manipulating this cache with the optimistic response and previously it don't have it, the it never trigger it